lib/ansible/plugins/terminal/nxos.py

```diff
@@ -70,18 +74,22 @@ def on_become(self, passwd=None):
 
         cmd = {u'command': u'enable'}
         if passwd:
-            cmd[u'prompt'] = to_text(r"(?i)[\r\n]?Password: $", errors='surrogate_or_strict')
+            cmd[u'prompt'] = to_text(
+                r"(?i)[\r\n]?Password: $", errors='surrogate_or_strict')
             cmd[u'answer'] = passwd
             cmd[u'prompt_retry_check'] = True
 
         try:
-            self._exec_cli_command(to_bytes(json.dumps(cmd), errors='surrogate_or_strict'))
+            self._exec_cli_command(
+                to_bytes(json.dumps(cmd), errors='surrogate_or_strict'))
             prompt = self._get_prompt()
             if prompt is None or not prompt.strip().endswith(b'enable#'):
-                raise AnsibleConnectionFailure('failed to elevate privilege to enable mode still at prompt [%s]' % prompt)
+                raise AnsibleConnectionFailure(
+                    'failed to elevate privilege to enable mode still at prompt [%s]' % prompt)
         except AnsibleConnectionFailure as e:
             prompt = self._get_prompt()
-            raise AnsibleConnectionFailure('unable to elevate privilege to enable mode, at prompt [%s] with error: %s' % (prompt, e.message))
+            raise AnsibleConnectionFailure(
```

Comment:
The formatting seems to be off on this file. Is it black or yapf?

Reply:
I noticed it too. I use autopep8 and this didnt throw any error. So didnt change it.